articles/the-development-of-5g-protocol-standardization/index.md

```diff
@@ -0,0 +1,178 @@
+
+
+ ### Overview
+This article will cover:
+- [Introduction to 5G protocol standardization](#Introduction-to-5G-protocol-standardization)
+- [5G networking mode](#5G-networking-mode)
+- [Development of uRLLc and mMTC](#Development-of-uRLLc-and-mMTC)
+- [5G Network security](#5G-Network-security)
+---
+
+
+1. ### Introduction to 5G protocol standardization.
+- #### 3GPP (3rd Generation Partnership Project) Releases.
+
+3GPP comprises of several Releases, among which are;
+
+-	R99.
+-	Rel-4 to Rel-16.
+-	The newly proposed Rel-17 and Rel-18.
+
+Rel-14 and all the initial releases define the previous mobile networks such as 4G, 3G, 2G, and 1G. LTE is defined from Rel-8, LTE-A is from Rel-10, and LTE-A Pro is from Rel-12, to mention just but few recent releases.
+
+- #### 3GPP 5G Releases.
+
+5G is defined in 3GPP Release 15 (Rel-15) and Release 16 (Rel-16), which constitute the following:
+
+1. NextGen Core (NGC) network.
+2. New Radio (NR).
+3. LTE Advanced Pro Evolution.
+4. EPC Evolution.
+
+Among the 5G, new technologies are New Radio and NextGen Core network.
+
+Other technologies that have been improved from some of the preceding Releases of 3GPP are EPC Evolution and LTE Advanced pro Evolution. 
+
+Below is the description of the Releases of 5G:
+
+#### Rel-15
+It is popularly considered the basic version of 5G.
+
+It is phase 1 of the 5G system that implemented the following improvement on NR;
+
+*Construct the NR technical framework*
++	New waveform - the F-OFDM technology is used.
++	Coding modulation and channel.
++	Massive MIMO (Multiple Input Multiple Output) - supports up to 64T64R.
++	Numerology, frame structure - refers to the change of the timeslot length and frame structure caused by different subcarrier spacing.
++	Flexible duplex - the uplink and downlink configurations are flexible. Also, the uplink and downlink can be included in the same timeslot.
+
+*Network architecture ready*
++	Non-standalone/Standalone.
++	Uplink and downlink decoupling.
++ CU-DU high-level segmentation.
+
+*Industry basic design*
++	URLLc (Ultra-Reliable Low Latency Communication).
+
+The implementation of the 5G protocol in Rel-15 underwent a transition through stages, which was split into three stages, as outlined below:
+
+**Early Rel-15 drop:** This stage focused on the third architecture option, also known as non-standalone NR (NSA NR). It employed the use of an LTE-A system of LTE base stations (called eNB) added to NR base stations (called gNB) and an Evolved Packet Core network (EPC) without any involvement of 5G core network (NGC). This phase of the Rel-15 standard was frozen in Dec 2017.
+
+<<<<<<< HEAD
```

Comment:
These strings appear in between. Pull the changes made by me, and then push the changes on your end. This is due to an error in using Git on your local editor.